README.md

```diff
@@ -189,6 +189,18 @@ The EncryptedCardData is encrypted according to the OAEP padding scheme. Once de
   }
 }
 
+### Failed callback
+
+In case that the user rejects the payment and the payment times out, a failed callback is sent to the FailedPaymentCallbackUrl url supplied in the initiate payment call.
```

Comment:
Did you mean 'or the payment times out'?

Reply:
In the flow diagram it is "and", but there might be other cases.